test/higherorderfns.jl

```diff
@@ -10,7 +10,7 @@ using Test
 using SparseArrays
 using LinearAlgebra
 using Random
-include("forbidproperties.jl")
+include(joinpath(@__DIR__, "forbidproperties.jl"))
```

Comment:
This seems unnecessary complication, since `include` is already relative to `@__DIR__`

Reply:
see this: https://github.com/JuliaSparse/SparseArrays.jl/pull/417#issuecomment-1646678186

Reply:
The workflow already checks that the correct version is loaded tho. See https://github.com/JuliaSparse/SparseArrays.jl/blob/main/.ci/test_and_change_uuid.jl

Reply:
by workflow I meant local workflow, not github actions